README.md

```diff
@@ -106,3 +106,41 @@ To run all the unit tests, run:
 To run a specific test or set of tests, use the `GINKGO_FOCUS` argument.
 
 	make test GINKGO_FOCUS="component function tests"
+
+### Making temporary changes to components the operator manages
+
+The operator creates and manages resources and will reconcile them to be in the desired state. Due to the
```

Comment:
If we're going to document this here, we need to shout loud and clear that this is intended for development only and is not to be used in production clusters. Setting this annotation and using custom configuration can and will lead to unexpected and potentially catastrophic behavior during upgrade, etc.